meson.build

```diff
@@ -41,6 +41,10 @@ if get_option('debug')
     tinyxml_extra_args += '-DTINYXML2_DEBUG'
 endif
 
+if host_machine.system() == 'linux'
```

Comment:
Meson does this for us, so we don't need it here.

Reply:
Gotcha, removed it. Another reason I like meson!